code/analyze/ac/stats/de_StatsFreqPreferences.m

```diff
@@ -51,6 +50,12 @@
           if si == 1
             stats.(prop).mean = cell(n_sigmas, 1);
             stats.(prop).std = cell(n_sigmas, 1);
+            if strcmp(prop, 'freq') % This code is used for sigma vs xover computations
```

Comment:
Mmm.. I'm too sleepy to figure this out, and it's a very minor style thing. So let's pass :)